### PR TITLE
Neutral armies

### PR DESCRIPTION
**Pull request by tttppp**
This is needed for https://github.com/zond/godip/pull/40

Hopefully this fixes the issue with spaces in nation names.

### CODE REVIEW

File: app/js/dippymap.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ function dippyMap(container) {
...
+	that.contrastNeutral = "#ffffffff";
```
Comment (tttppp) [inline app/js/dippymap.js]: We could try transparent instead - ffffff00 gives:

westernworld901transparent

**Comment by zond** [comment]
Cool!

I haven't fully grasped the changes you're doing for this new variant, but I haven't seen anything I object to.

Merge when your feel ready :)

**Comment by zond** [comment]
Just remember to pull both godip and goaeoas as well, before running the diplicity tests and deploying :)

**Comment by tttppp** [comment]
This definitely fixes one problem with multiword nations. There might be others too!

Merging this as it shouldn't break anything. For WW901 I also need to include the variant as beta. I'll do that separately when it's ready.

**Pull request merged**